### PR TITLE
Selected items can now be deleted with 1 click G3 2024 v5 #15278

### PR DESCRIPTION
**Pull request by a22edvbu**
I've added the option to deleted selected items with one click. When clicking the checkboxes on one or more items on the coursepage and then clicking the delete button, all selected items will be deleted.

### CODE REVIEW

**Review by a22moamy — CHANGES_REQUESTED** [review]
Avoid using "var" in code. Use "let" or "const" instead.

**Review by a22moamy — APPROVED** [review]
Code looks good to me, haven't tested the functionality though

File: DuggaSys/sectioned.js [hunk 1]
```diff
@@ -704,10 +704,11 @@ function confirmBox(operation, item = null) {
     $("#sectionShowConfirmBox").css("display", "flex");
     $('#close-item-button').focus();
   } else if (operation == "deleteItem") {
-    deleteItem(active_lid);
+    //deleteItem(active_lid);
```
Comment (b22marah) [inline DuggaSys/sectioned.js:707]: remove

File: DuggaSys/sectioned.js [hunk 2]
```diff
@@ -784,10 +785,10 @@ function markedItems(item = null) {
         var tempKind = $(this).parents('tr').attr('value');
         if (tempDisplay != "none" && (tempKind == "section" || tempKind == "moment" || tempKind == "header")) {
           itemInSection = false;
-          //console.log("loop breaker: "+tempItem);
+          console.log("loop breaker: "+tempItem);
```
Comment (b22marah) [inline DuggaSys/sectioned.js:788]: remove

File: DuggaSys/sectioned.js [hunk 3]
```diff
@@ -784,10 +785,10 @@ function markedItems(item = null) {
...
         } else {
           subItems.push(tempItem);
-          //console.log("added: "+tempItem);
+          console.log("added: "+tempItem);
```
Comment (b22marah) [inline DuggaSys/sectioned.js:791]: remove

File: DuggaSys/sectioned.js [hunk 4]
```diff
@@ -796,53 +797,53 @@ function markedItems(item = null) {
...
-  if (hideItemList.length == 0) {
+  if (selectedItemList.length == 0) {
     // Disable ghost button when no checkboxes is checked
     document.querySelector('#hideElement').disabled = true;
     document.querySelector('#hideElement').style.opacity = 0.7;
     hideVisibilityIcons();
 
   }
-  console.log(hideItemList);
+  console.log(selectedItemList);
```
Comment (b22marah) [inline DuggaSys/sectioned.js:846]: remove

File: DuggaSys/sectioned.js [hunk 5]
```diff
@@ -1001,13 +1002,17 @@ function prepareItem() {
...
 //----------------------------------------------------------------------------------
 
-function deleteItem(item_lid = null) {
-  lid = item_lid ? item_lid : $("#lid").val();
-  item = document.getElementById("lid" + lid);
-  item.style.display = "none";
-  item.classList.add("deleted");
+function deleteItem(item_lid = []) {
+  for (var i = 0; i < item_lid.length; i++) {
+    console.log("deleteItem: " + item_lid[i]);
```
Comment (b22marah) [inline DuggaSys/sectioned.js:1007]: remove

**Review by b22marah — CHANGES_REQUESTED** [review]
Remove all dead code / comments & console.logs, otherwise code looks great, and fulfills the requirements, good job! :)

**Review by b22marah — APPROVED** [review]
LGTM!!

**Pull request merged**